lua/nvim_treesitter_playground_query_omnifunc.lua

```diff
@@ -0,0 +1,37 @@
+local query_linter = require 'nvim-treesitter-playground.query_linter'
+
+local M = {}
+
+function M.omnifunc(findstart, base)
+  if findstart == 1 then
+    local start = vim.fn.col('.') - 1
+    local result = vim.fn.matchstrpos(vim.fn.getline('.'):sub(1, start), '\\v("|\\\\|\\-|\\w)*$')
+    return result[2]
+  end
+
+  local buf = vim.api.nvim_get_current_buf()
+  local query_lang = query_linter.guess_query_lang(buf)
+
+  local ok, parser_info = pcall(vim.treesitter.inspect_language, query_lang)
+  parser_info = ok and parser_info
+
+  if parser_info then
```

Comment:
Instead of that, maybe :

```suggestion
  if ok then
```